deployment/plat-app-services/sads-offline/base.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: sads-offline
+  name: sads-offline
+spec:
+  ports:
+  - name: http
+    port: 8501
+    protocol: TCP
+    targetPort: 8501
+  selector:
+    app: sads-offline
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: sads-offline
+  name: sads-offline
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sads-offline
+  template:
+    metadata:
+      labels:
+        app: sads-offline
+    spec:
+      containers:
+        - image: "ghcr.io/souayb/sads_offline:main"
```

Comment:
is there any chance Ginkgo could actually release an image with a version number, e.g. `0.2.0`, that corresponds to a git tag with the exact same name containing the code the image was built from. that way if there's an issue we can easily trace it back to the original source code the image was built from.

also, if they release from main in the future, we'll have to remember to kill the pod, otherwise K8s won't download the new image b/c it's got the same tag name...

alternatively we could set the image pull policy to always pull, but that's sort of wasteful...

Reply:
Good morning. 

Good spot just updated the mesh-infra/argocd/projects/plat-app-services/kustomization.yaml to include sads-offline (included in this review) https://github.com/c0c0n3/kitt4sme.live/pull/222/commits/e096dc4fdd159ea2c6701c152577a083dc79d588 

As for the docker image tag, ill send an email and have that resolved. Just to note i still need to do some port forwarding locally to access sads-offline/ on my device, I think we can confirm this issue after merge and resolve it then.

Lets wait to hear back about the tag and update that before merging.

Reply:
hey I think it's quite possible the problem you were having on your box trying to access sads offline could be related to the rewrite rule in the Istion routing---which you now have deleted. you can retest your local setup after we merge this pr.

re: image tag, @souayb released a versioned image, if you could please update the k8s image tag accordingly, thanks!!